### DISCUSSION TITLE
MDX page not found

### DISCUSSION THREAD

**Discussion by Cha-Shao** [vercel/next.js#72143] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
npx --no-install next info
warn  - Failed to fetch latest canary version. (Reason: fetch failed.)
      Detected "14.2.15". Visit https://github.com/vercel/next.js/releases.     
      Make sure to try the latest canary version (eg.: `npm install next@canary`) to confirm the issue still exists before creating a new issue.

Learn more: https://nextjs.org/docs/messages/opening-an-issue

Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 11 Enterprise        
  Available memory (MB): 32509
  Available CPU cores: 20
Binaries:
  Node: 20.13.1
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.2.15
  eslint-config-next: 14.2.15
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.6.3
Next.js Config:
  output: N/A
```

### Which example does this report relate to?

app-dir-mdx

### How are you deploying your application? (if relevant)

next dev

### Describe the Bug

all mdx page not found

### Expected Behavior

mdx pages should render as usual

### To Reproduce

https://github.com/Cha-Shao/mdx-test

**Comment by samcx** [comment]
@Cha-Shao You should be on v15 (15.0.2) if you do a fresh install of that example. I was not able to replicate the issue on the latest.
